README.md

```diff
@@ -1 +1,6 @@
-# ReviewProject
\ No newline at end of file
+# ReviewProject
+FCC Test Suite Template
+
+Fork this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? Click hear to learn how to do it.
```

Comment:
Consider adding more information in your README file and adding your personal links to Github, Linkdin and Twitter